tidy3d/components/simulation.py

```diff
@@ -3569,6 +3572,41 @@ def snap_to_grid(geom: Geometry, axis: Axis) -> Geometry:
             snapped_center = snap_coordinate_to_grid(self.grid, center, axis)
             return set_bounds(geom, (snapped_center, snapped_center), axis)
 
+        def is_normal_volumetric_equivalent_accurate(
```

Comment:
This was just a quick placeholder for checking if the materials are conductors which is based on some given value for conductivity.